sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/CloudPubsubTransforms.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.io.gcp.pubsublite;
+
+import static com.google.cloud.pubsublite.cloudpubsub.MessageTransforms.fromCpsPublishTransformer;
+import static com.google.cloud.pubsublite.cloudpubsub.MessageTransforms.toCpsPublishTransformer;
+import static com.google.cloud.pubsublite.cloudpubsub.MessageTransforms.toCpsSubscribeTransformer;
+
+import com.google.cloud.pubsublite.Message;
+import com.google.cloud.pubsublite.SequencedMessage;
+import com.google.cloud.pubsublite.cloudpubsub.KeyExtractor;
+import com.google.pubsub.v1.PubsubMessage;
+import io.grpc.StatusException;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.PTransform;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.PCollection;
+
+// A class providing transforms between Cloud Pub/Sub and Pub/Sub Lite message types.
+public final class CloudPubsubTransforms {
+  private CloudPubsubTransforms() {}
+
+  // Transform a collection of SequencedMessages to Cloud Pub/Sub received PubsubMessages.
+  public static PTransform<PCollection<? extends SequencedMessage>, PCollection<PubsubMessage>>
+      toCpsSubscribeTransform() {
```

Comment:
The names of these methods aren't very clear.  I think something like PubsubLiteConversions.sequenceMessageToCloudPubsubMessage would read better.  The distinction between publish and subscribe isn't obvious from the method name anyways, and is probably better in comments.

Reply:
Will drop this method.